src/mf/mf-backend.cpp

```diff
@@ -311,30 +327,8 @@ namespace librealsense
                         if( p_hdr->dbch_devicetype != DBT_DEVTYP_DEVICEINTERFACE )
                             break;
                         auto data = reinterpret_cast<extra_data*>(GetWindowLongPtr(hWnd, GWLP_USERDATA));
-                        auto next = data->_last;
-                        std::wstring temp = reinterpret_cast<DEV_BROADCAST_DEVICEINTERFACE*>(lParam)->dbcc_name;
```

Comment:
Any idea what was the reason for all of this code?
And why it is not needed anymore?

Reply:
I can't very well comment about the first.
But it's not needed any more because we just get everything again fresh (after a delay), just as if we did query_devices. This is how udev does it, too.

Reply:
Probably they were trying to save time...